Add Devant concepts docs (#21)
wso2/docs-devant: en/docs/devant-concepts/environments.md

# Environments

Devant offers developers one or more environments to run their integrations within a given data plane. By default, the Devant cloud data plane provides two environments (i.e., development and production). Each project in Devant is associated with one or more environments available in the organization. For example, project A may choose to utilize dev, staging, and production environments, while project B may only use development and production environments.

> You can promote integrations within a project across available environments. When you promote an integration, its configuration values can be overridden with environment-specific values.

Reviewer: ```suggestion
You can promote integrations within a project across available environments. Promoting an integration allows you to override its configuration values with environment-specific values.
```
